src/ILCompiler.MetadataTransform/tests/PrimaryMetadataAssembly/Platform.cs

```diff
@@ -140,3 +140,19 @@ public IndexerNameAttribute(String indexerName)
     }
 }
 
+namespace Internal.Reflection
```

Comment:
Can the new stuff go to a separate file? We have multiple `Platform.cs` files for our unit tests and I would like to do something about it at some point.